server/bin/unittests

```diff
@@ -334,6 +334,10 @@ function _setup_state {
         exit $res
     fi
 
+    # Ensure proper expected umask.
+    find ${_testopt} ${_testopt_sat} -perm -u=rx -exec chmod 775 {} \;
```

Comment:
If you really mean to reset *directory* permissions, shouldn't you use `find [...] -type d` rather than just counting on the `-perm` filter?

Reply:
This resets both executables and directories.

Reply:
Then I was just confused regarding your *intent* from the wording in the commit message: "To get around this, all directories are reset via `chmod 775`, and all non executable files are reset via `chmod 664`, and all executable
files are reset via `chmod 775`."